pywavefront/cache.py

```diff
@@ -0,0 +1,171 @@
+"""
```

Comment:
You did a whole lot of work here. That is pretty awesome 👏 

Can you add some unit tests for this code? Here are some really quick examples:
* parse a sample file and verify correct results
* parse a cache file with an error
* either meta or bin file are missing
* meta file doesn't match bin file
* bin file is not a gzip file

I'll add a few more interspersed comments for potential test cases.